plugin/trino-redshift/src/test/java/io/trino/plugin/redshift/TestRedshiftConnectorTest.java

```diff
@@ -101,6 +103,40 @@ protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
         }
     }
 
+    @Test
+    public void testSuperColumnType()
```

Comment:
reminder that CI doesn't run this test class at all (not even with /test-with-secrets).